styles/sidebar/sidebar.css

```diff
@@ -1,17 +1,25 @@
 #sidebar {
-  margin: 1rem;
-  color: var(--primary-color);
   min-width: 15rem;
-  height: 80vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: space-evenly;
-   background: var(--default);
-  border: 3px solid var(--secondary-color);
   border-radius: 3%;
+  background: var(--default);
```

Comment:
Better to use variable name to be clear for background, eg. default-bg

Reply:
sure, will do that